### ISSUE TITLE
Refactor navbar menu so links show on desktop view

### ISSUE THREAD

**Issue by briancodex** [briancodex/community-website#59]
Need to refactor the navbar so the nav links display on desktop views instead of just a hamburger menu. You can switch it the a hamburger menu at the break point of 768px.

**Comment by abhinavmaharana** [comment]
can i do it. you want whole menu in the navbar but at mobile you want hamburger.

**Comment by briancodex** [comment]
yeah basically show all the options on desktop, then only display hamburger when you reach tablet view or lower, so 768px or below.
